dev/com.ibm.ws.microprofile.metrics.3.0_fat_tck/fat/resources/logging.properties

```diff
@@ -0,0 +1,78 @@
+#Logging file used by com.ibm.ws.microprofile.metrics11.tck.launcher.FATSuite
+#Thu, 14 Mar 2019 14:27:11 -0400
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
```

Comment:
2020?

Reply:
This file is "copied". I believe we don't change the copyright for such files since it isn't _new_.